chart/values.yaml

```diff
@@ -0,0 +1,237 @@
+# Default values for kube-httpcache.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: quay.io/mittwald/kube-httpcache
+  pullPolicy: IfNotPresent
+  tag: "stable"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+# Enable StatefulSet or Deployment
+statefulset:
+  enabled: true
+
+deployment:
+  enabled: false
+
+# Enable configMap for Varnish Template File (see below defaultConfig)
+# OR use extravolume with name "template" if the file is too big
+configmap:
+  enabled: true
+
+# kube-httpcache specific configuration
+cache:
+  # name of frontend service
+  frontendservice: kube-httpcache
```

Comment:
Now, the service is deployed with `{{ include "kube-httpcache.fullname" . }}` as name, whereas the `{{ .Values.cache.frontendservice }}` name is still used in some places to reference it.

Reply:
fixed